### PR TITLE
POL-193 - add minimum spend to billing center cost anomaly 

### PR DESCRIPTION
**Pull request by Chasein96**
### Description

Updated policy to include a minimum spend requirements that billing centers must have to be detected.

### Issues Resolved

Resolved reporting on small amount increases.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by cdwilhelm** [comment]
@Chasein96   please check the danger errors.

**Comment by Chasein96** [comment]
Screen Shot 2020-02-11 at 5 08 45 PM

File: cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt [hunk 1]
```diff
@@ -6,13 +6,11 @@ long_description ""
 severity "high"
 category "Cost"
 tenancy "single"
-default_frequency "daily"
-info(
-  version: "1.3",
+
+info( version: "1.2",
```
Comment (cdwilhelm) [inline cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt]: You need to bump the version, it should be 1.4 to match the change log.

File: cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt [hunk 2]
```diff
@@ -6,13 +6,11 @@ long_description ""
 severity "high"
 category "Cost"
 tenancy "single"
-default_frequency "daily"
```
Comment (cdwilhelm) [inline cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt]: Is there a reason you removed the default_frequency?

Comment (Chasein96) [reply cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt]: I did not mean to remove that.

**Pull request merged**